api-reference/user-app/vitals/vital-information-table.mdx

```diff
@@ -0,0 +1,20 @@
+Here is the information regarding various vitals:
+
+| **Vital Name**                                    | **LOINC Code** | **Measurement Unit**      | **MCUM UCUM Code**   | **EKA ID**        |
+|:--------------------------------------------------|:---------------|:--------------------------|:----------------------|:-------------------|
+| **Min Heart Rate**                              |                | BPM (beats per minute)     |                      |                   |
```

Comment:
make this `Min Heart Rate in 24 hrs` -- as the loinc code is DIFFERENT from Min heart rate, same for Max and average . Once dr Rajshree gives it also reverify on Google yourself to make sure its right.

Reply:
add the base `heart rate` vital too